lib/matplotlib/ticker.py

```diff
@@ -855,17 +878,19 @@ def set_locs(self, locs):
             vmax = math.log(vmax) / math.log(b)
             numdec = abs(vmax - vmin)
 
-        if numdec > 1:
+        if numdec > self.minor_thresholds[0]:
             # Label only bases
-            self.sublabel = set((1,))
-        else:
+            self._sublabels = set((1,))
+        elif numdec > self.minor_thresholds[1]:
             # Add labels between bases at log-spaced coefficients
-            c = np.logspace(0, 1, (4 - int(numdec)) + 1, base=b)
-            self.sublabel = set(np.round(c))
+            c = np.logspace(0, 1, b//2 + 1, base=b)[1:-1]
+            self._sublabels = set(np.round(c))
+        else:
+            self._sublabels = set(np.linspace(2, b-1, b-2))
 
     def __call__(self, x, pos=None):
         """
-        Return the format for tick val `x` at position `pos`.
+        Return the format for tick val `x`.
         """
         b = self._base
```

Comment:
No, I thought it was misleading.  The `pos` value is never used in this Formatter, and doesn't have to be specified.